app/domain/authentication/authn_oidc/fetch_oidc_secrets.rb

```diff
@@ -0,0 +1,58 @@
+require 'command_class'
+require 'conjur/fetch_required_secrets'
+
+module Authentication
+  module AuthnOidc
+
+    # TODO: (later version) Fix CommandClass so we can add errors directly
+    # inside of it
+    #
+    # TODO: list any OIDC or other errors here.  The errors are part of the
```

Comment:
Do we have issues for these `TODO`s so that they don't get forgotten?

Reply:
created an issue for the first TODO and removed it.

The second one is just wrong, i removed it. we have another place to list OIDC errors